packages/amplify-category-auth/src/__tests__/commands/enable.test.js

```diff
@@ -1,3 +1,5 @@
+const { FeatureFlags } = require('amplify-cli-core');
+FeatureFlags.getBoolean = () => false;
```

Comment:
nit move this line to after the require statements (and in other files as well)

Reply:
:+1: